README.md

```diff
@@ -59,6 +59,30 @@ Use `^C` to stop the container and `docker-compose down` to remove the last used
 Many TypeScript and static file changes can be made without re-building the image,
 but as necessary, rebuild the image using Step 1.
 
+#### Step By Step
+
+Explicit steps for setting up CCM in a development environment.
```

Comment:
Maybe indicate somewhere that this is just for configuration, not execution? You're not repeating the Docker commands. Maybe this could be moved into the **Configuration** section? I'm okay whatever you decide.